app/src/main/java/com/douglasbruce/tasky/features/agenda/AgendaScreen.kt

```diff
@@ -81,7 +131,66 @@ internal fun AgendaScreen(
                 ),
             )
         },
-        containerColor = MaterialTheme.colorScheme.primary,
+        floatingActionButton = {
+            Box(
+                modifier = Modifier
+                    .minimumInteractiveComponentSize()
+                    .wrapContentSize(Alignment.TopStart)
+            ) {
+                FloatingActionButton(
+                    containerColor = Black,
+                    contentColor = White,
+                    onClick = { expanded2 = !expanded2 },
+                ) {
+                    Icon(imageVector = TaskyIcons.Add, contentDescription = null)
+                }
+                DropdownMenu(
+                    expanded = expanded2,
+                    onDismissRequest = { expanded2 = false },
+                ) {
+                    DropdownMenuItem(
+                        text = {
+                            Text(
+                                text = stringResource(R.string.event),
+                                style = TextStyle(
+                                    fontSize = 16.sp,
+                                    lineHeight = 15.sp,
+                                    fontWeight = FontWeight(400),
+                                )
+                            )
+                        },
+                        onClick = { /*TODO*/ },
+                    )
+                    DropdownMenuItem(
+                        text = {
+                            Text(
+                                text = stringResource(R.string.task),
+                                style = TextStyle(
+                                    fontSize = 16.sp,
+                                    lineHeight = 15.sp,
+                                    fontWeight = FontWeight(400),
+                                )
+                            )
+                        },
+                        onClick = onAddTaskClick,
+                    )
+                    DropdownMenuItem(
+                        text = {
+                            Text(
+                                text = stringResource(R.string.reminder),
+                                style = TextStyle(
+                                    fontSize = 16.sp,
+                                    lineHeight = 15.sp,
+                                    fontWeight = FontWeight(400),
+                                )
+                            )
+                        },
+                        onClick = { /*TODO*/ },
+                    )
```

Comment:
Could think about moving this into a separate composable and just pass the text and onClick lambda. Currently, changing just the text style would require 3 changes

Reply:
Missed refactoring this in PR#16. Will do so in PR#17